src/Dfe.PlanTech.Web/Controllers/PagesController.cs

```diff
@@ -15,11 +14,15 @@ public PagesController(ILogger<PagesController> logger)
         _logger = logger;
     }
 
-    public async Task<IActionResult> Index([FromServices] GetPageQuery getPageQuery)
+    [HttpGet("/{route?}")]
+    public async Task<IActionResult> GetByRoute(string? route, [FromServices] GetPageQuery query)
     {
```

Comment:
null checks for warnings?

Reply:
The warning isn't valid; there is a null check on line 20, but linting isn't picking up for some reason.